fe/src/main/java/org/apache/doris/qe/ShowExecutor.java

```diff
@@ -624,10 +624,14 @@ private void handleShowColumn() throws AnalysisException {
                                                         aggType));
                         }
                     }
+                } else {
+                    throw new AnalysisException("Table " + db.getFullName() + "." + showStmt.getTable() + " no found");
```

Comment:
Better to use ErrorReport and ErrorCode to unify error message and error code. And this code can be written as 

```
ErrorReport.reportDdlException(ErrorCode.ERR_BAD_TABLE_ERROR, tableName);
```